### PR TITLE
Generate new SDK for ManagedServiceIdentity with new swagger api version

### PR DESCRIPTION
**Pull request by wendyNEU**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
https://github.com/Azure/azure-rest-api-specs/pull/4492
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
@wendyNEU 
- Please fix the PR description/title
- Looks like no new code was generated. Please create a generate.ps1 file similar to [this](https://github.com/Azure/azure-sdk-for-net/blob/master/src/SDKs/Advisor/Management.Advisor/generate.ps1) and regenerate the code.

**Comment by wendyNEU** [comment]
Hi dsgouda, generate.cmd already exist, do we still need generate.ps1?

**Comment by dsgouda** [comment]
@wendyNEU yes, we are moving all RPs to using generate.ps1 to generate SDKs

File: src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Generated/ManagedServiceIdentityClient.cs [hunk 1]
```diff
@@ -287,7 +328,7 @@ private void Initialize()
             Operations = new Operations(this);
             UserAssignedIdentities = new UserAssignedIdentitiesOperations(this);
             BaseUri = new System.Uri("https://management.azure.com");
-            ApiVersion = "2015-08-31-preview";
```
Comment (dsgouda) [inline src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Generated/ManagedServiceIdentityClient.cs]: @wendyNEU are you planning to release a stable version for your api version `2015-08-31-preview` ?

File: src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Microsoft.Azure.Management.ManagedServiceIdentity.csproj [hunk 2]
```diff
@@ -5,8 +5,8 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.ManagedServiceIdentity</PackageId>
-    <Description>Provides developers with libraries for ManagedSericeIdentity under Azure Resource manager to perform CRUD operations on identities.</Description>
-    <Version>0.10.0-preview</Version>
+    <Description>Provides developers with libraries for ManagedServiceIdentity under Azure Resource manager to perform CRUD operations on identities.</Description>
+    <Version>0.11.0</Version>
     <AssemblyName>Microsoft.Azure.Management.ManagedServiceIdentity</AssemblyName>
     <PackageTags>Microsoft Azure resource management;msi;managedserviceidentity;</PackageTags>
     <PackageReleaseNotes>Taking dependency on 10.0.3 version of Newtonsoft nuget package.</PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Microsoft.Azure.Management.ManagedServiceIdentity.csproj]: Please update the `PackageReleaseNotes` here

File: src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Microsoft.Azure.Management.ManagedServiceIdentity.csproj [hunk 3]
```diff
@@ -5,8 +5,8 @@
 
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.ManagedServiceIdentity</PackageId>
-    <Description>Provides developers with libraries for ManagedSericeIdentity under Azure Resource manager to perform CRUD operations on identities.</Description>
-    <Version>0.10.0-preview</Version>
+    <Description>Provides developers with libraries for ManagedServiceIdentity under Azure Resource manager to perform CRUD operations on identities.</Description>
```
Comment (dsgouda) [inline src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Microsoft.Azure.Management.ManagedServiceIdentity.csproj]: Please remove description, use `PackageReleaseNotes` instead

File: README.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 
 
-# Microsoft Azure SDK for .NET
+## All PRs should be opened against master branch. This branch is being retired and will be deleted.
```
Comment (dsgouda) [inline README.md]: We have already defaulted our repo to `master` branch, don't think we need this change. Appreciate the help

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Generated/ManagedServiceIdentityClient.cs (hunk 1 above)
Comment (wendyNEU) [reply src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Generated/ManagedServiceIdentityClient.cs]: I plan to release a stable version for api version 2018-11-30

**Comment by wendyNEU** [comment]
It seems the test is broken,  will take a look and send another commit to fix it.

**Pull request closed without merging**